internal/pkg/cli/svc_deploy.go

```diff
@@ -425,6 +426,12 @@ func (o *deploySvcOpts) deploySvc(addonsURL string) error {
 
 	if err := o.svcCFN.DeployService(conf, awscloudformation.WithRoleARN(o.targetEnvironment.ExecutionRoleARN)); err != nil {
 		o.spinner.Stop(log.Serrorf("Failed to deploy service.\n"))
+		errors, describeErr := o.svcCFN.GetStackErrors(conf)
+		if describeErr != nil {
+			return fmt.Errorf("describe stack: %w", describeErr)
+		}
+		log.Infoln("Cloudformation status reason:")
```

Comment:
nit: should we check if the length of `errors` is 0? and if so just returning the error without a reason